SofascoreAcademy/SofascoreAcademy/View/HeaderView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  HeaderView.swift
+//  SofascoreAcademy
+//
+//  Created by Anton Pomper on 26.03.2024..
+//
+
+import UIKit
+import SnapKit
+import SofaAcademic
+
+protocol HeaderViewDelegate: AnyObject {
+    func didTapSettings()
+}
+
+class HeaderView: BaseView {
+    
+    private let logoImageView = UIImageView()
+    private let trophyImageView = UIImageView()
+    private let settingsImageView = UIImageView()
+    
+    weak var delegate: HeaderViewDelegate?
+    
+    override func addViews() {
+        super.addViews()
+        addSubview(logoImageView)
+        addSubview(trophyImageView)
+        addSubview(settingsImageView)
+    }
+    
+    override func styleViews() {
+        super.styleViews()
+        
+        backgroundColor = .colorPrimaryDefault
+        
+        logoImageView.contentMode = .scaleAspectFit
+        logoImageView.clipsToBounds = true
+        
+        trophyImageView.contentMode = .scaleAspectFit
+        trophyImageView.clipsToBounds = true
+        
+        settingsImageView.contentMode = .scaleAspectFit
+        settingsImageView.clipsToBounds = true
+    }
+    
+    override func setupConstraints() {
+        super.setupConstraints()
+        
+        logoImageView.snp.makeConstraints() { make in
+            make.top.equalToSuperview().offset(14)
+            make.leading.equalToSuperview().inset(16)
+            make.width.equalTo(132)
```

Comment:
znam da u dizajnu piše da je širina slike 132, ali u velikoj veličini slučajeva to ne radimo. S obzirom da će slika pokušati zadržati svoj aspect ratio, a postavljena je visina, ovaj constraint je nepotreban